packages/frontend-defaults/report.api.md

```diff
@@ -43,4 +44,13 @@ export interface CreateAppOptions {
 export function createPublicSignInApp(options?: CreateAppOptions): {
   createRoot(): React_2.JSX.Element;
 };
+
+// @public (undocumented)
+export function getAvailableFeatures(config: Config): FrontendFeature[];
+
+// @public (undocumented)
+export function resolveFeatures(options: {
```

Comment:
let's workshop these names a bit.

For the first one I'm thinking this kinda thing:

- `getDiscoveredFeatures`
- `discoverAvailableFeatures`
- `resolveConfiguredFeatures`

For the second:

- `resolveAsyncFeatures`
- `resolveFeatureLoaders`

For both of them let's wrap up the `FrontendFeature[]` as `{ features: FrontendFeature[] }` too. Both of these are lower level plumbing so we don't need to ergonomic benefits of avoiding that wrapping.

Reply:
Went with `discoverAvailableFeatures()` and `resolveAsyncFeatures()`.